README.md

```diff
@@ -619,6 +621,131 @@ When enabled, the overall patient-level results will contain the raw results of
 ];
 ```
 
+## Composite Measures
+
+:warning: Composite measure support is highly experimental, and may change as the [specification](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html) evolves :warning:
+
+`fqm-execution` supports the calculation of [composite measures](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html) under the following conditions:
+
+- The provided measure bundle **must** contain only one composite `Measure` resource (i.e. a FHIR `Measure` resource with `composite` as the scoring code)
+  - See the [conformance requirements of a composite measure](https://build.fhir.org/ig/HL7/cqf-measures/composite-measures.html#component-quality-measures) and the [CQFMCompositeMeasure profile](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-composite-measure-cqfm.html) for specific guidance regarding a composite `Measure` resource
+- The provided measure bundle **must** contain all component `Measure` resources and their dependent `Library` resources needed for calculation
```

Comment:
I also tested removing one of the component measures from the bundle even though it was still a related artifact in the composite measure. Do we want to throw an error in this case? When I tested, it did not throw an error, the detailedResults just did not included population results for that component.

Reply:
Looks like the measure score gets calculated correctly, though.

Reply:
https://github.com/projecttacoma/fqm-execution/pull/235/commits/c34809c7218c1e0cf54d30b17744cdffe7850bb8